### ISSUE TITLE
Version 0.4.0 doesn't scan Redhat images - error: Unknown OS

### ISSUE THREAD

**Issue by sfoxdev** [aquasecurity/trivy#353]
Using Trivy v0.4.0
```
docker run --rm aquasec/trivy:0.4.0 --no-progress sonatype/nexus3:3.20.1
2019-12-30T20:08:08.786Z        INFO    Need to update DB
2019-12-30T20:08:08.786Z        INFO    Downloading DB...
2019-12-30T20:08:11.211Z        INFO    Reopening DB...
2019-12-30T20:08:18.805Z        FATAL   error in image scan: failed to scan the image: failed to analyze OS: Unknown OS
```
Using Trivy v0.3.1
```
docker run --rm aquasec/trivy:0.3.1 --no-progress sonatype/nexus3:3.20.1
2019-12-30T20:08:29.052Z        INFO    Need to update DB
2019-12-30T20:08:31.038Z        INFO    Reopening DB...
2019-12-30T20:08:41.462Z        INFO    Detecting RHEL/CentOS vulnerabilities...

sonatype/nexus3:3.20.1 (redhat 8.1)
===================================
Total: 142 (UNKNOWN: 0, LOW: 11, MEDIUM: 111, HIGH: 17, CRITICAL: 3)
+------------------------+------------------+----------+--------------------+---------------+------------------------------------------------+
|        LIBRARY         | VULNERABILITY ID | SEVERITY | INSTALLED VERSION  | FIXED VERSION |                     TITLE                      |
+------------------------+------------------+----------+--------------------+---------------+------------------------------------------------+
| avahi-libs             | CVE-2017-6519    | MEDIUM   | 0.7-19.el8         |               | avahi: Multicast DNS responds                  |
|                        |                  |          |                    |               | to unicast queries outside of                  |
|                        |                  |          |                    |               | local network                                  |
+                        +------------------+          +                    +---------------+------------------------------------------------+
|                        | CVE-2018-1000845 |          |                    |               | avahi: DNS amplification                       |
|                        |                  |          |                    |               | and reflection to spoofed                      |
|                        |                  |          |                    |               | addresses                                      |
+------------------------+------------------+----------+--------------------+---------------+------------------------------------------------+
| bzip2-libs             | CVE-2019-12900   | HIGH     | 1.0.6-26.el8       |               | bzip2: out-of-bounds write in                  |
|                        |                  |          |                    |               | function BZ2_decompress                        |

```

**Comment by knqyf263** [comment]
v0.4.1 fixed this issue. I'm sorry, but would you please check it?

**Comment by sfoxdev** [comment]
Fixed in Trivy v0.4.1

**Comment by knqyf263** [comment]
Thank you for the confirmation 🙏
